certified-connectors/envoy/apiDefinition.swagger.json

```diff
@@ -2708,7 +2708,7 @@
                               "type": "object",
                               "properties": {
                                 "type": {
-                                  "type": "string",
+                                  "type": "object",

```

Comment:
@iannater The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. can you please confirm that no existing flows and users will be affected by this change?

Reply:
@Amjed-Ayoub Yes! This field is currently breaking existing flows and we are making this change to fix it.